NOnion/Network/TorGuard.fs

```diff
@@ -383,7 +383,7 @@ type TorGuard private (client: TcpClient, sslStream: SslStream) =
                             CellNetInfo.Time =
                                 DateTimeUtils.ToUnixTimestamp DateTime.UtcNow
                             OtherAddress = otherAddress
-                            MyAddresses = List.singleton netInfo.OtherAddress
+                            MyAddresses = List.empty
```

Comment:
@aarani nit: use List.Empty, not List.empty

Reply:
Fixed